core/string/ustring.cpp

```diff
@@ -3074,6 +3074,11 @@ int String::rfindn(const String &p_str, int p_from) const {
 }
 
 bool String::ends_with(const String &p_string) const {
+	int l = p_string.length();
```

Comment:
Please also use `l` further down instead of recalculating the `p_string.length()`.
(Make the changes with `git commit --amend` and `git push --force` to keep it all in one commit, see [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request)).

Reply:
Done!